packages/bpk-component-scrollable-calendar/stories.js

```diff
@@ -18,14 +18,30 @@
 /* @flow */
 
 import React from 'react';
+
 import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
 
-import BpkScrollableCalendar, { BpkScrollableCalendarDate } from './index';
+import { weekDays, formatMonth, formatDateFull } from './test-utils';
+import BpkScrollableCalendar, {
+  BpkScrollableCalendarDate,
+  BpkScrollableCalendarGrid,
+} from './index';
 
 storiesOf('bpk-component-scrollable-calendar', module)
   .add('Default', () => <BpkScrollableCalendar />)
```

Comment:
Could we remove this story as it is just the default example one included in the boilerplate?

Reply:
Other than that, LGTM  🎉 

I'll leave @shaundon to review as he knows more about this than I do 👍

Reply:
I think it's ok to leave for now, under the assumption that it'll be fleshed out before this is released. If it's still there by the time we get close to 1.0.0 we can address it.

Reply:
I actually asked Boris to keep this in.

Why would we remove it? We should always have a default story.

Reply:
I think we've just crossed wires - the story itself is fine, it's just that the component underneath that it displays just says "This is an example component".

When I said "_it's ok to leave for now, under the assumption that it'll be fleshed out before this is released_", I meant the component underneath.

So in summary I'm good with it!

Reply:
Ah gotcha - yeah I mean this whole component is a work in progress so before it gets released we can tie up stuff like this.